src/resources/auth/controllers/AuthController.ts

```diff
@@ -15,28 +19,84 @@ const authTokenExpiresIn = process.env.AUTH_TOKEN_EXPIRES_IN || "1m";
 
 @Service()
 export class AuthController {
-	constructor(public usersService: UsersService) {}
+	constructor(
+		public usersService: UsersService,
+		public organizationsService: OrganizationsService,
+	) {}
 
 	async login(req: express.Request, res: express.Response) {
 		if (req.user) {
 			const authUser = req.user as AuthUser;
-			const token = jwt.sign(authUser, jwtSecret, {
-				expiresIn: authTokenExpiresIn,
-			});
+
+			const accessTokens: AccessToken[] = this.generateAccessTokens(authUser);
 			const user = await this.usersService.readById(authUser.identifier);
-			if (!user) {
-				return res.status(404).send(new ErrorResponseBuilder().notFoundResponse("User not found").build());
+			const organizations: Organization[] = await this.getOrganizationsForMemberships(user);
```

Comment:
Ich würde die neuen Zeilen (`const accessTokens` und `const organizations`) in den `if`-Zeig unten schieben, weil nichts davon nötig ist, wenn es eh keinen `user` gibt. Dann kann der null-Check in `getOrganizationsForMemberships` auch weg und der Code wird simpler.